ComponentKit/DataSources/Common/CKComponentAnnouncerBaseInternal.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <vector>
+
+@interface CKComponentAnnouncerBase()
+{
+@public
+  // we make this public, so it can be accessed from CK::AnnouncerHelper
```

Comment:
nit: I realize this predates your changes here, but it'd be nice to make this comment match our style for multiline...